### PR TITLE
Filter tab for used cars and new cars

### PR DESCRIPTION
**Pull request by vatrey**
[Filter Tab (Used and New Cars)](https://abhirams.atlassian.net/browse/P3ECWFC-44?atlOrigin=eyJpIjoiMTgzNmRmNmZjODRhNDdjNDg4M2NhNGE3ZDNhMmJhNGEiLCJwIjoiaiJ9)

### CODE REVIEW

File: src/pages/CarsList/CarsList.tsx [hunk 1]
```diff
@@ -8,26 +8,29 @@ import { CarDetails } from "../../models/CarDetails";
 import { carTypeList } from "../../models/CarType";
```
Comment (pant-manas) [inline src/pages/CarsList/CarsList.tsx]: Resolve conflicts

File: src/pages/CarsList/CarsList.tsx [hunk 2]
```diff
@@ -44,45 +45,71 @@ function CarsList() {
...
     }
+    switch(usedParam) {
+      case "true":
+        setIsUsed(true);
+        break;
+      case "false":
+        setIsUsed(false);
+        break;
+      default:
+        setIsUsed(undefined);
```
Comment (pashedin) [inline src/pages/CarsList/CarsList.tsx]: Please, do not ever set undefined intentionally, if you wnat to specify an empty or nullish value always go for null.

**Review by pashedin — CHANGES_REQUESTED** [review]
All looks good just replace undefined with null

**Review by pashedin — CHANGES_REQUESTED** [review]
All looks good just replace undefined with null

**Review by pashedin — APPROVED** [review]

**Pull request merged**